packages/wp-now/src/constants.ts

```diff
@@ -0,0 +1,43 @@
+import os from 'os';
+import path from 'path';
+
+/**
+ * The hidden folder name for storing WP Now related files.
+ */
+export const WP_NOW_HIDDEN_FOLDER = '.wp-now';
+
+/**
+ * The full path to the hidden WP Now folder in the user's home directory.
+ */
+export const WP_NOW_PATH = path.join(os.homedir(), WP_NOW_HIDDEN_FOLDER);
+
+/**
+ * The path where WordPress zip files and unzipped folders will be stored within the WP Now folder.
+ */
+export const WORDPRESS_ZIPS_PATH = path.join(WP_NOW_PATH, 'wordpress_zips');
+
+/**
+ * The file name for the SQLite plugin name.
+ */
+export const SQLITE_FILENAME = 'sqlite-database-integration';
+
+/**
+ * The full path to the SQLite database integration folder.
+ */
+export const SQLITE_PATH = path.join(WP_NOW_PATH, `${SQLITE_FILENAME}-main`);
+
+/**
+ * The URL for downloading the latest version of WordPress.
+ */
+export const WP_DOWNLOAD_URL = 'https://wordpress.org/latest.zip';
+
+/**
+ * The URL for downloading the SQLite database integration WordPress Plugin.
```

Comment:
in several of these, I think that if we are referring to the "SQLite database integraton" plugin where that's it's full name, it could help to surround it in quotes to make it clear we're referring to a specific name, and not some SQLite _database integration_

Reply:
Done ! https://github.com/WordPress/wordpress-playground/pull/188/commits/1504b5f74767aff0b4aa7e1719fe2d224ae48e93